src/app/tl-drop-down/tl-drop-down.component.ts

```diff
@@ -9,7 +9,7 @@ import { TamuAbstractBaseComponent } from '../shared/tl-abstract-base.component'
 export class TlDropDownComponent extends TamuAbstractBaseComponent {
 
   /** Used to override the drop-down menu active button. */
-  @Input() btnType = 'success';
+  @Input() btnType = '';
```

Comment:
should this be `undefined` by default instead of `''`

Reply:
Resolved this.